### PR TITLE
Feature: Glanceable UI theme + additional progress percentage. 

### PR DESCRIPTION
**Pull request by thegarbz**
This PR further addresses issue #749 for a more progress centric UI. 

It includes a new "Glanceable" theme which is a variant on the standard theme but with a horizontal progress bar. 

Additionally a second progress percentage element has been added to the app nested with the horizontal progress bar. This allows a progress percentage to be visible and in two different places on the screen depending on which progress bar is shown. The CSS is setup to ensure this second progress bar is hidden unless enabled by a theme.

This change was tested against both the default theme as well as the NOX theme to ensure it doesn't break either.

### CODE REVIEW

**Comment by thegarbz** [comment]
Sorry please excuse the "Merge branch 'master' " messages. It took me a while to figure out how to do rebase pulls from within Visual Studio.

File: src/app/job-status/job-status.component.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
       <div class="job-info__progress-bar__wrapper">
         <div class="job-info__progress-bar" [style.width]="job.progress + '%'"></div>
       </div>
+      <div class="job-info__progress-bar-percentage">{{ job.progress }}<span style="font-size: 75%">%</span></div>
```
Comment (UnchartedBull) [inline src/app/job-status/job-status.component.html:22]: I'll have a look whether we can somehow work around the third percentage element (might take a little while).

File: themes/theGarbz/Glanceable/custom-styles.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
+
+/**** Override Hidden Blocks ****/
+
+/***[hidden]{
+    display:block!important;
+}**/
```
Comment (UnchartedBull) [inline themes/theGarbz/Glanceable/custom-styles.css:6]: please remove if not needed anymore

File: themes/theGarbz/Glanceable/custom-styles.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    color:white!important;
+    text-shadow: 1px 1px 4px black;
+}*/
+
+
+
+
+
+
+
```
Comment (UnchartedBull) [inline themes/theGarbz/Glanceable/custom-styles.css:113]: there should only be a single empty line at the bottom of each file.

**Comment by UnchartedBull** [comment]
Should work now without an additional percentage diff. Let me know if everything works as you expect.

**Review by UnchartedBull — APPROVED** [review]

**Comment by thegarbz** [comment]
Cheers I'll have a look tonight if this achieved what was intended.

**Review by thegarbz — COMMENTED** [review]
This works great. I didn't know about the "+" element selector in CSS. This solution is much neater than an additional element. 

Thanks.

**Comment by UnchartedBull** [comment]
Yeah I also just learned about that selector a week or so ago.

**Pull request merged**